### PR TITLE
Test tweets

### PR DESCRIPTION
**Pull request by CesarCachay**
Adding some methods of the model Tweets (index, show)

### CODE REVIEW

File: app/controllers/tweets_controller.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#Tweets Required attributes: id, user_id, text, likes(number), replies(number)
+class TweetsController < ApplicationController
+  def index
+    render json: Tweet.where(tweet_id = nil), status: :ok
```
Comment (Linzeur) [inline app/controllers/tweets_controller.rb]: Another way would be "tweet_id => nil", but I like it!

File: db/schema.rb [hunk 2]
```diff
@@ -34,7 +34,6 @@
   end
 
   create_table "tweets", force: :cascade do |t|
-    t.string "name"
```
Comment (Linzeur) [inline db/schema.rb]: Thank you for removing the field that it's unnecessary

File: app/controllers/tweets_controller.rb (hunk 1 above)
Comment (CesarCachay) [reply app/controllers/tweets_controller.rb]: Also, I do like the way you wrote it, It's such a good practice!

File: db/schema.rb (hunk 2 above)
Comment (CesarCachay) [reply db/schema.rb]: Yes, that field belongs to user table

**Pull request merged**